wiki/Beatmaps/es.md

```diff
@@ -0,0 +1,60 @@
+# Beatmaps
+
+Los beatmaps son los niveles a batir en [osu!](/wiki/osu!_glossary), similares a los de los [Juegos de DS] (/wiki/Glossary/#ds-games).Los beatmaps se componen de una canción y varios [hit objects](/wiki/hit_objects] situados en el espacio y el tiempo al ritmo de la música. Pueden tener, como fondo, una imagen, un vídeo o un [storyboard](/wiki/storyboard).
```

Comment:
`[hit objects](/wiki/hit_objects]` -> `[hit objects](/wiki/hit_objects)`
there is a syntax error: change the last square bracket to a round bracket